archive/r/racket/README.md

```diff
@@ -5,6 +5,7 @@ Welcome to Sample Programs in Racket!
 ## Sample Programs
 
 - [Hello World in Racket](https://therenegadecoder.com/code/hello-world-in-racket/)
+- [Fibonacci in Racket]
```

Comment:
Could you, please, add a link to this line. I assume we don't have an article, yet. In that case we usually link to a create article issue. In this case it would be "Add Fibonacci in Racket Article". If the issue doesn't exist yet, feel free to create it.

Reply:
hi,
I will definitely add the text in place of links tomorrow, for all my PRs/

thanks for looking into my PRs.

cheers

Reply:
Hi,
If you can please check the links to article issues added in README.md

cheers